modules/apple_support/0.12.1/source.json

```diff
@@ -0,0 +1,4 @@
+{
+    "integrity": "sha256-xgS3WGXAf0WCjH//1f2/+BQVqehKaPcanB2OOyaU5Uc=",
```

Comment:
I calculated `sha256-9t8AfuoSKBgBrWym0XBzu9/pXSm1c4u9byVGYRyDOxw=`

Reply:
Where are the instructions on how to calculate this? I nabbed the code from the python to do it

Reply:
See https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
Hmm, the code from python should do it correctly, not sure what went wrong.

Reply:
What files specifically are you running this on?

```
% cat apple_support.0.12.1.tar.gz | openssl dgst -sha256 -binary | openssl base64 -A
xgS3WGXAf0WCjH//1f2/+BQVqehKaPcanB2OOyaU5Uc=
```

This usage matches what I had here

Reply:
Oh, I see, the url is wrong, it should be 0.12.1

Reply:
Sorry about that! I had to bump mid way through